lib/classes/eik-config.js

```diff
@@ -30,8 +30,7 @@ module.exports = class EikConfig {
     }
 
     get type() {
-        const valid = validate.type(this[_config].type);
-        return valid.value;
+        return this[_config].type || 'package';
```

Comment:
Is it worth extracting this and the `out` default into a constant? Could also pull them from the schema with something like:

```js
const schema = require("../schemas/eikjson.schema.json")
const DEFAULTS = Object.keys(schema.properties)
  .reduce((acc, key) => {
    const prop = s.properties[key]; 
    if(prop.hasOwnProperty("default")) {acc[key] = prop.default}
    return acc;
  }, {})
  ```

Reply:
Good point! Not sure why I didn't think of using the schema directly.